packages/lib-bitcoinjs/index.ts

```diff
@@ -81,6 +117,10 @@ export class DfnsWallet implements SignerAsync {
 
     return Psbt.fromHex(res.signedData.replace(/^0x/, ''))
   }
+
+  public tweakSigner(taprootMerkleRoot: Buffer): SignerAsync {
+    return new DfnsWallet(this.metadata, { ...this.options, taprootMerkleRoot })
```

Comment:
I prefer an anonymous class that's locked down to only do schnorr signing and nothing else. see my previous comment why I don't like to return a wallet. tweaked signer does not map to a dfns wallet.